src/lib/net/util/identifiers.ts

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable roblox-ts/lua-truthiness */
+
+import { RunService } from "@rbxts/services";
+
+export namespace Identifiers {
+	let autoSerde: Folder | undefined;
```

Comment:
Can we name this variable something else?

Reply:
Yep.